test/index.test.js

```diff
@@ -34,22 +34,64 @@ describe('markdown-it-image-figures', function() {
     assert.strictEqual(res, expected);
   });
 
-  it('should add convert alt text into a figcaption when opts.figcaption is set', function() {
+  it('should convert title text into a figcaption when opts.figcaption is true', function() {
```

Comment:
Just cleaned up the test names to better match what they're doing.

Reply:
Great cleanup!